test-integration/pom.xml

```diff
@@ -11,10 +11,10 @@
 
     <properties>
         <!-- Dependencies versions -->
-        <version.infinispan>12.0.0.Dev04</version.infinispan>
+        <version.infinispan>11.0.3.Final</version.infinispan>
```

Comment:
Why Infinispan version need to be downgraded for master?

Reply:
That's unintentional mistake created by working on tests for 2.0.x and checking out the directory to match. I'll fix that.